test/apicoverage.sh

```diff
@@ -31,12 +31,14 @@ readonly SERVING_TEST_DIR=$(dirname $0)
 readonly APICOVERAGE_IMAGE=$SERVING_TEST_DIR/apicoverage/image
 readonly APICOVERAGE_TOOL=$SERVING_TEST_DIR/apicoverage/tools
 
+function knative_setup() {
+  install_knative_serving
+}
+
 # Script entry point.
 initialize $@
 
-header "Setting up environment"
-
-install_knative_serving || fail_test "Knative Serving installation failed"
+failed=0
```

Comment:
where is this used?

Reply:
I followed the e2e_tests.sh script which sets this flag. The API coverage script does not use it, but not sure if its not required for the go_test call below.

Reply:
I see, this is not needed in this case, please delete it and I will lgtm

Reply:
Done.